content/en/docs/concepts/storage/storage-classes.md

```diff
@@ -413,7 +413,7 @@ parameters:
     * Distribute volume: `volumetype: none`
 
     For available volume types and administration options, refer to the
-    [Administration Guide](https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3.1/html/Administration_Guide/part-Overview.html).
+    [Administration Guide](https://access.redhat.com/documentation/en-US/Red_Hat_Storage/3.1).
```

Comment:
I think this is a better link as it does not lock into a specific version and forwards to the latest version, atm is 3.5
```suggestion
    [Administration Guide](https://access.redhat.com/documentation/en-us/red_hat_gluster_storage/).
```

Reply:
Suggestion given by @reylejano  looks better